File: Frontend/SkySync/src/app/models/horas-vuelo.ts
export interface HorasVuelo {
  id?: number;
  usuarioId: number;
  fecha: string;
  horas: number;
  tipoVuelo: 'Local' | 'Nacional' | 'Internacional';
  origen?: string;
  destino?: string;
  notas?: string;
}

File: SkySync_Solutions/SkySync.Domain/Entidades/HorasVuelo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SkySync.Domain.Entidades
{
    public class HorasVuelo
    {
        [Key]
        public int Id { get; set; }

        [Column("UsuarioId")]
        public int UsuarioId { get; set; }

        [Column("Fecha")]
        public DateTime Fecha { get; set; }

        [Column("Horas")]
        public decimal Horas { get; set; }

        [Column("TipoVuelo")]
        public string TipoVuelo { get; set; } = "Local";

        [Column("Origen")]
        public string? Origen { get; set; }

        [Column("Destino")]
        public string? Destino { get; set; }

        [Column("Notas")]
        public string? Notas { get; set; }
    }
}
